### ISSUE TITLE
Can I use src import on SFC custom block?

### ISSUE THREAD

**Issue by Ocyss** [posva/unplugin-vue-router#269]
Because I wrote a lot of `route.meta` values, my `.vue` file is very long, and I don’t change route.meta often, so I plan to split it and put it in a separate `yaml` file. Moreover, it bothers me that the yaml in sfc is highlighted but not formatted, so I think it would be more convenient and elegant to put it in a separate `yaml` file.

But I found that route sfc does not support src to import a file. Can you help me?

![image](https://github.com/posva/unplugin-vue-router/assets/68412205/c7fd31fa-2607-4c2e-b88a-edbddf7f57dd)

**Comment by posva** [comment]
I don’t think it makes sense to extract the route into a different file because one of the points of using this is to have the route information next to the page component

**Comment by Ocyss** [comment]
> I don’t think it makes sense to extract the route into a different file because one of the points of using this is to have the route information next to the page component

What you said makes sense, but my routing files are arranged according to `folder/index.vue`, so I want to put the routing information files together in the same folder.
